content/app-dev/create-actor/run.md

```diff
@@ -5,4 +5,59 @@ weight: 5
 draft: false
 ---
 
-TBD
\ No newline at end of file
+### Pre-Requisites
+
+In this guide we're going to start the actor "the long way" so that you can get a feel for all of the moving parts of the process. Our tooling documentation should help you get actors started easier once you've been through this guide.
+
+### Deploy a Local Docker Registry
+
+The `wash` tooling and the embedded REPL launch actors and capability providers from OCI (Open Container Initiative)-compliant registries. One such registry is the docker registry. You can run this locally with the following command:
+
+```
+docker run -d -p 5000:5000 --restart=always --name registry registry:2
+```
+
+For more information on running the registry, you can read [Docker's documentation](https://docs.docker.com/registry/deploying/).
+
+### Push Your Actor
+
+Next you'll want to push your actor to the local registry. If you haven't configured authentication or any other restrictions, you should be able to use the following `wash` command to push it to the registry:
+
+```
+wash reg push --insecure localhost:5000/(image):(tag) ./target/wasm32-unknown-unknown/debug/(actor).wasm
+```
+
+For example, you might use `newactor` and `v1` as the image and tag respectively, giving you a local registry URL (you'll need to remember this) of `http://localhost:5000/newactor:v1`. Make sure the path you supply here is to the _signed_ version of your actor's WebAssembly module.
+
+### Launch the Actor
+
+There are countless ways to run the actor we just created, but the easiest is probably to use our **REPL**, by typing `wash up` at the command line. You should see a screen that looks similar to the following:
+
+![Wash Up](../wash_up.png)
+
+Once inside the wash REPL, you can issue this command to start the actor (make sure you change the OCI URL to the one you used in the previous step):
+
+```
+start actor localhost:5000/newactor:v1
+```
+
+Next, we'll need to start the HTTP Server. Fortunately, the wasmCloud official HTTP server capability provider is published in Azure Container Registry, so you can start it with the following REPL command:
+
+```
+start provider https://wasmcloud.azurecr.io/http_server:0.15.0
+```
+
+With both the provider and the actor running, the only thing left to do is _link_ the two. This provides a set of configuration values that is unique to one actor's use of a provider. Type the following command to link your actor with the provider:
+
+```
+link ... ???? ??? ??
```

Comment:
This should be
```
ctl link (ACTOR_MODULE_KEY) VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M wasmcloud:httpserver PORT=8080
```
Perhaps we can also throw in a snippet like:

If you had your module key autogenerated via `wash` when you ran `make`, you can find that key by running the following command in the REPL:
```
claims inspect --insecure localhost:5000/newactor:v1
```
Under the `Module` section, you'll see a 56 character public key starting with **M**. That is your actor's module key that you can use with the previous `link` command.